tests/flow/random_graph.py

```
@@ -0,0 +1,262 @@
+
+
+def random_double():
+    return random()
+
+
+ALL_VALUES_EXCEPT_ARRAY = [random_string, random_int,
+                           random_bool, random_double]
+
+
+def random_array():
+    l = randint(0, 10)
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)